### PR TITLE
init Stereo

### PR DESCRIPTION
**Pull request by mancoast**
@revmischa are you OK with these headers?

### CODE REVIEW

File: src/projectM-sdl/pmSDL.cpp [hunk 1]
```diff
@@ -144,7 +167,10 @@ void projectMSDL::keyHandler(SDL_Event *sdl_evt) {
         case SDLK_f:
             if (sdl_mod & KMOD_LGUI || sdl_mod & KMOD_RGUI || sdl_mod & KMOD_LCTRL) {
                 // command-f: fullscreen
-                toggleFullScreen();
+				// Stereo requires fullscreen
+#ifndef STEREOSCOPIC_SBS
```
Comment (revmischa) [inline src/projectM-sdl/pmSDL.cpp]: maybe this check would be better in `toggleFullScreen()` - just make it always toggle to true or no-op if stereo

File: src/projectM-sdl/projectM_SDL_main.cpp [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+* Authors: Created by Mischa Spiegelmock on 6/3/15.
+*
+* 
+* experimental Stereoscopic SBS driver functionality by
+*	RobertPancoast77@gmail.com
+*/
 
 #include "pmSDL.hpp"
 
-#define FAKE_AUDIO          0
```
Comment (revmischa) [inline src/projectM-sdl/projectM_SDL_main.cpp]: why remove? FAKE_AUDIO is at least useful for testing. is it documented anywhere else?

Comment (mancoast) [reply src/projectM-sdl/projectM_SDL_main.cpp]: I moved theses definitions to the header file so they can be effective across multiple files.

File: src/projectM-sdl/pmSDL.cpp (hunk 1 above)
Comment (mancoast) [reply src/projectM-sdl/pmSDL.cpp]: Ok i will rework these

**Pull request merged**